ome2024-ngff-challenge/src/Main.svelte

```diff
@@ -173,12 +173,12 @@
     <div class="summary">
       <h3 style="text-align:center">
         {#if csvUrl == SAMPLES_HOME}
-          This site makes
           <strong style="font-weight:600">{filesizeformat(totalBytes)}</strong>
-          of public bioimage data available in the Zarr v3 format.
-          The OME-NGFF community converted and now hosts
-          this federated collection of the next version OME-Zarr.
+          of public bioimage data in the Zarr v3 format.
+          converted and now hosted by the federated OME-NGFF community
```

Comment:
Upper-case `Converted..`  Maybe remove the "now"? Not sure what that means?  "Converted and hosted by the...

Reply:
Several new versions pushed. You might look at the history to find several different ones.